From: Tomas Henzl <thenzl@redhat.com>
Date: Sun, 29 Aug 2010 15:49:01 -0400
Subject: [block] cciss: remove double setting of h->busy
Message-id: <1283097002-3341-3-git-send-email-thenzl@redhat.com>
Patchwork-id: 27851
O-Subject: [RHEL6 PATCH 02/63] cciss: Removed double setting of h->busy
	configuring to 0. Once is enough.
Bugzilla: 568830
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Removed double setting of h->busy configuring to 0.  Once is enough.

diff --git a/drivers/block/cciss.c b/drivers/block/cciss.c
index b46304d..f32f21e 100644
--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -2063,7 +2063,6 @@ freeret:
 	return -1;
 mem_msg:
 	printk(KERN_ERR "cciss: out of memory\n");
-	h->busy_configuring = 0;
 	goto freeret;
 }
 
